prow/git/v2/client_factory.go

```diff
@@ -285,6 +279,9 @@ func (c *clientFactory) ClientFor(org, repo string) (RepoClient, error) {
 		if err := os.MkdirAll(cacheDir, os.ModePerm); err != nil && !os.IsExist(err) {
 			return nil, err
 		}
+		if err := authenticateCookieFile(c.cookieFilePath, repoClient); err != nil {
```

Comment:
Sorry, but if we have just one callsite let's not make a func for it - can just inline here